### DISCUSSION TITLE
Player Indicators - True/False active flag for "active in Wilderness only"

### DISCUSSION THREAD

**Discussion by rossmas** [runelite/runelite#16094] [Q&A] [answered]
Suggestion to add a checkbox toggle feature to enable "wilderness only" or "active everywhere" player indicators.

Commonly players currently manually toggle hide others on/off going in to/out of the wild. 

Hope you can make this huge QoL change and give greater safety to those who enjoy the plugin, but maybe sometimes forget to turn it off going to the wild.

**Comment by Enriath** [comment]
This is not something we're gonna do. See #3684

**Reply by rossmas** [reply]
If it was a raised as a PR, would you accept it. Or is the addition of 1 more useful toggle just too much?

**Reply by rossmas** [reply]
Can you explain WHY you won't do something like this? The thread you linked seemed to imply that since you can re-configure the plugin when entering and exiting the wilderness, that there is no issue. 

The issue is a QoL one. Not a functional one

**Reply by Nightfirecat** [reply]
See: https://github.com/runelite/runelite/pull/14468#issuecomment-993881791

**Reply by rossmas** [reply]
So, can I raise a PR for it then? Since its something "you" dont want to do? 

Surely, 1 toggle checkbox that will get used often, won't be useless bloat to RL....

**Reply by Enriath** [reply]
I would recommend you actually read the comment that Night linked.

**Reply by rossmas** [reply]
Ok, so no new features can be added because might go down the rabbit hole of adding a feature for everything? Whats the threshold for adding a new feature then?

**Reply by Nightfirecat** [reply]
You can do whatever you like, but as that linked comment thread says, I don't think we're interested in adding such a specific toggle. If you're interested in developing a more generalized solution (as my comment in that thread mentions, in the vein of #2686), we'd be happy to discuss that. :)

**Reply by rossmas** [reply]
I see - I am personally not interested in developing a general hotkey toggle plugins/features on/off as I do not think thats a good solution, although I understand your intention with it. 

All I'm asking is for the wildy option mentioned specifically for player indicators (a great plugin btw). Would that be acceptable as a raised PR? or only the more general solution youre referring to in #2686 ?

**Reply by Enriath** [reply]
As we have said now multiple times, we're not interested in adding an incredibly specific toggle like this.

**Reply by Nightfirecat** [reply]
I understand you want this simple toggle added to the plugin, but per Adam's comment, the issue on our end is not that it's easy to add it, it's that it becomes one more thing for us to maintain, and given all the edge cases and other pvp activity that players would likely want to see included (aside from just wilderness), it's not something we want to maintain. I'd encourage you to run a development build if it's a change you want to use for yourself, as that plugin's code is entirely open source.

**Reply by rossmas** [reply]
Sure, may try for myself, since it seems it would never get added to RL even if it worked well I guess... My third attempt to contribute to RL/plugins and always rejected Sadge

**Comment by vancwo** [comment]
This feature request is getting attention on /r/2007scape here: https://old.reddit.com/r/2007scape/comments/100jcx0/to_whoever_made_this_plugin_really_appreciate_it/

**Reply by rossmas** [reply]
its something I wanted for a long time, but seeing all the support encouraged me to come to the source

**Comment by Adam-** [comment, marked as answer]
I don't mind adding this if it is extremely popular. I still agree with the sentiment in my comment from Dec 2021 https://github.com/runelite/runelite/pull/14468#issuecomment-993881791. Ideally we would have a cleaner way to do this like configuration presets that aren't so specific to one feature of one plugin. But since we don't, I can add a toggle to the plugin.

**Reply by rossmas** [reply]
Thanks for your response. I understand that it could be a slippery slope and it is indeed a simple toggle for users currently, however, I would argue that this feature alone doesn't really lead down that path. I agree that no pvp/multi combat/target world nonsense need be considered for adding in the future because it is a grey area and would be a pain.

I think this stands apart from the slippery slope issues though. Anyway, happy new year :)

**Reply by Nightfirecat** [reply]
Added in c428728707b0edb62ceffe1c44aa9b6fecbd252e
